data/json/mutations/mutations.json

```diff
@@ -3106,7 +3106,7 @@
     "points": 6,
     "vitamin_cost": 210,
     "visibility": 10,
-    "ugliness": 12,
+    "ugliness": 2,
     "description": "Your skin has warped into a highly protective but inconsistent medley of chitin, scales, fur, and bone.",
```

Comment:
The whole vibe of Chimera is a discoherent mishmash of animal features in a single amalgamation. It should be very ugly, and become more so as the features become more pronounced.